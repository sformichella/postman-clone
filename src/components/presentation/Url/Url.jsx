import React from "react";

export default function Url() {
  return (
    <input id="url" type="text" defaultValue="URL"/>
  );
}
