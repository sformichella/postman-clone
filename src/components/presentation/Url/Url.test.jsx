import React from "react";
import { render, cleanup } from "@testing-library/react";
import Url from "./Url";

describe("Url component", () => {
  afterEach(() => cleanup());
  it("renders Url", () => {
    const { asFragment } = render(<Url />);
    expect(asFragment()).toMatchSnapshot();
  });
});
