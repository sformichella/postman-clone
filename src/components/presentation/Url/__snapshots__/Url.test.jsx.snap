// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Url component renders Url 1`] = `
<DocumentFragment>
  <label
    for="url"
  />
  <input
    id="url"
    name="url"
    placeholder="URL"
    type="text"
  />
</DocumentFragment>
`;
